src/App.jsx

```diff
@@ -14,57 +16,56 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
```

Comment:
Rename `value` to `selectedGood` to better reflect its purpose.

```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```